test/unit/app/data-fetchers/tequilapi-proposal-fetcher.spec.js

```diff
@@ -54,14 +55,14 @@ describe('TequilapiProposalFetcher', () => {
   let clock
   const interval = 1001
   const tequilapi = new IdentityTequilapiClientMock([
-    new ProposalDTO({
-      id: '0x1',
+    parseProposalDTO({
+      id: 1,
       providerId: '0x1',
       serviceType: 'openvpn',
       serviceDefinition: {}
     }),
-    new ProposalDTO({
-      id: '0x2',
+    parseProposalDTO({
+      id: 2,
```

Comment:
TIL: these ids are not globally unique - unique proposal id is `id`+`providerId`+`serviceType`.
Would suggest to leave this id non-unique intentionally to make this more clear.

```suggestion
      id: 1,
```